src/components/Project/ProjectHeader/BookmarkProjectButton.tsx

```diff
@@ -29,14 +37,31 @@ const BookmarkProjectButton = () => {
     }
   }
 
+  const unbokmarkProject = async () => {
+    return
+  }
+
   return (
-    <Tooltip title={t`Bookmark Project`} placement="bottom">
-      <Button
-        onClick={() => bookmarkProject()}
-        icon={<StarOutlined />}
-        type="text"
-      />
-    </Tooltip>
+    <>
+      {isLoading && <></>}
```

Comment:
**suggestion** maybe slap the loading icon from antd here

Reply:
Can do this but imo would look a little weird to have a small loader in the actions bar.

Maybe the unfilled star icon if we're fetching?